Common/src/extension/groovy/io/github/groovymc/cgl/api/extension/ItemExtensions.groovy

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (C) 2022 GroovyMC and contributors
+ * SPDX-License-Identifier: LGPL-3.0-or-later
+ */
+
+package io.github.groovymc.cgl.api.extension
+
+import groovy.transform.CompileStatic
+import net.minecraft.core.BlockPos
+import net.minecraft.core.Position
+import net.minecraft.nbt.CompoundTag
+import net.minecraft.tags.TagKey
+import net.minecraft.world.item.Item
+import net.minecraft.world.item.ItemStack
+import net.minecraft.world.item.crafting.Ingredient
+import net.minecraft.world.level.ItemLike
+import net.minecraft.world.phys.Vec3
+import org.codehaus.groovy.runtime.DefaultGroovyMethods
+
+@CompileStatic
+class ItemExtensions {
+    static ItemStack count(ItemLike self, int count) {
+        new ItemStack(self, count)
+    }
+
+    static ItemStack multiply(ItemLike self, int count) {
+        new ItemStack(self, count)
+    }
+
+    static ItemStack multiply(Integer count, ItemLike item) {
+        new ItemStack(item, count)
+    }
+
+    static ItemStack count(ItemStack self, int count) {
+        self.setCount(count)
+        return self
+    }
+
+    static ItemStack tag(ItemStack self, CompoundTag tag) {
```

Comment:
The purpose of the methods is to allow for the parantheses-less method calls and implicit calls such as `Items.X count 15 tag([yes: 16])`

Reply:
Makes sense; I'm tempted to say that maybe it's better to have this whole DSL as a category to avoid confusion in general usage, but maybe it doesn't matter that much